src/components/NavBar/NavBar.jsx

```diff
@@ -16,7 +16,7 @@ import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
```

Comment:
There shouldn't be any changes to this file in this PR that is scoped to solve the issue with the delete client modal. Has nothing to do with the Issue/PR at hand.